### PR TITLE
Adds a Cartfile target to LoopKit

### PR DESCRIPTION
**Pull request by rickpasetto**
A few months ago SwiftCharts was added as a Carthage dependency to LoopKit.  This target ensures carthage bootstrap gets run before the build.

### CODE REVIEW

File: LoopKit.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -4036,6 +4105,24 @@
 /* End PBXVariantGroup section */
 
 /* Begin XCBuildConfiguration section */
+		1D81903B25687FF2004D1BEC /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = 75U4X84TEG;
```
Comment (darinkrauss) [inline LoopKit.xcodeproj/project.pbxproj]: Please remove `DEVELOPMENT_TEAM` here and below. Thanks!

**Review by darinkrauss — CHANGES_REQUESTED** [review]
One change needed.

**Comment by rickpasetto** [comment]
@darinkrauss I made a few more changes than one :).  I also fixed the CI script and added caching to it.

**Review by darinkrauss — APPROVED** [review]
LGTM! 🚀

**Pull request merged**